marketplace/commercelayer-products/src/dataTransformer.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Transforms the API response of CommerceLayer into
+ * the product schema expected by the SkuPicker component
+ */
+export const dataTransformer = projectUrl => ({
+  id,
+  attributes: { image_url: imageUrl, name, code }
+}) => {
+  return {
+    id,
+    image: imageUrl,
+    name,
+    sku: code,
+    externalLink: `${projectUrl}/admin/skus/${id}/edit`
+  };
+};
+
+/**
+ * Transforms the API response of CommerceLayer into
+ * the product schema expected by SkuPicker Preview components
+ */
+export const previewDataTransformer = projectUrl => ({ id, imageUrl, name, code }) => {
```

Comment:
Why do these functions only differ slightly? It seems like they could potentially be merged.

Reply:
So the problem is that when you fetch data through a direct HTTP req the returned schema format differs from when you request the same data using their JS SDK. Doing both is unavoidable due to the way the Commerce Layer API implements pagination.

I explain more on that in the comments above the `fetchSKUs` function in index.js

Reply:
I unified them into 1 function.